bazel/README.md

```diff
@@ -515,6 +515,76 @@ debugging information files (`.dwp` files) and we can build a DWARF package file
 with `.dwp ` target. The `.dwp` file need to be presented in the same folder with the
 binary for a full debugging experience.
 
+# Debugging production core dumps
+
+In production, you'll likely be running a version of envoy compiled with `-c
+opt`. When we build envoy with `-c opt` it still has debug symbols attached
+which greatly bloats the binary size.
+Envoy uses fission which uses `gsplit-dwarf` to have the debug info partially in
+the `.o` but mostly in `.dwo` file for each object file.
+
+Most likely when you distribute the binary (or if you use the Envoy's
+release docker image) you will have stripped the debug information such that
+when you list the sections of the binary it might look as follows:
```

Comment:
how would you check that ?

Reply:
`size --format=SysV source/exe/envoy-static` would work, will include it in the document